plugins/txt/webportal/components/live-scene-txt.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+export default Component.extend({
+  gameApi: service(),
+  flashMessages: service(),
+  scenePose: '',
+  gameSocket: service(),
+  session: service(),
+  tagName: '',
+
+
+  actions: {
+
+    addTxt() {
+        let pose = this.get('scenePose');
+        if (pose.length === 0) {
+            this.get('flashMessages').danger("You haven't entered anything.");
+            return;
+        }
+        let api = this.get('gameApi');
+        this.set('scenePose', '');
+        api.requestOne('addTxt', { scene_id: this.get('scene.id'),
+            pose: pose }, null)
+        .then( (response) => {
+            if (response.error) {
+                this.get('flashMessages').error(response.error);
+                return;
+            }
+            this.scrollSceneWindow();
```

Comment:
This will throw a JS error.  There's no scroll method in this file to call.  On the up side, you probably don't need it.  The window should scroll when the live scene gets the new text 'pose'.

Reply:
Do I need to remove just line 30 here?

Reply:
Yeah